packages/terra-toggle-button/src/terra-dev-site/test/toggle-button/Animated.test.jsx

```diff
@@ -4,7 +4,7 @@ import ToggleButton from '../../../ToggleButton';
 
 const AnimatedToggleButton = () => (
   <ToggleButton id="animatedToggle" closedButtonText="Custom Text" isAnimated>
-    <p>Lorem ipsum dolor sit amet, <a id="link" href="#test">consectetur</a> adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
```

Comment:
Why the removal of the link here?

Reply:
I was receiving the following axe error:
[Skip-link](https://dequeuniversity.com/rules/axe/2.0/skip-link)

There a was no reason defined within the test for me to think it was necessary. Is it important to keep this link in here?

Reply:
It was intended to highlight that focusable elements can not be tabbed to when the toggle is closed but it looks like we don't have any wdio tests for it. I'm fine removing it for now as to not block the PR, but we should get a test in place for this in the future. I'll log an issue for the work.